src/extensions/core/load3d/Load3d.ts

```diff
@@ -621,6 +693,23 @@ class Load3d {
   }
 
   public remove(): void {
+    if (this.contextMenuHandlers) {
+      const canvas = this.renderer.domElement
+      canvas.removeEventListener(
+        'mousedown',
+        this.contextMenuHandlers.mousedown
+      )
+      canvas.removeEventListener(
+        'mousemove',
+        this.contextMenuHandlers.mousemove
+      )
+      canvas.removeEventListener(
+        'contextmenu',
+        this.contextMenuHandlers.contextmenu
+      )
```

Comment:
Could you use an [AbortController](https://kettanaito.com/blog/dont-sleep-on-abort-controller) instead?